style/client.css

```diff
@@ -3189,6 +3188,21 @@ a.ilink.yours {
 	background: #DDDDDD;
 }
 
+.dark .ps-popup {
+	position: absolute;
+	top: auto; right: auto; left: auto; bottom: auto;
+	text-align: left;
+	background: #0D0E0E;
+	color: #DDDDDD;
+	border: 1px solid #DDDDDD;
+	padding: 2px 10px;
+
+	border-radius: 6px;
+	box-shadow: inset 1px 1px 0 #ffffff, inset -1px -1px 0 #DDDDDD, 2px 2px 3px rgba(0,0,0,.2);
```

Comment:
@panpawn Your right I should have asked and will in the future and thanks for the edit never noticed that n_n